serialization/dotnet/text/Microsoft.Kiota.Serialization.Text.sln.sln

```diff
@@ -0,0 +1,28 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30114.105
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Kiota.Serialization.Text", "src\Microsoft.Kiota.Serialization.Text.csproj", "{29CEB2E3-93C5-4AFF-BAE1-DFF7BADF9A4D}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Kiota.Serialization.Text.Tests", "Microsoft.Kiota.Serialization.Text.Tests\Microsoft.Kiota.Serialization.Text.Tests.csproj", "{53A7CB1A-280A-4BEB-B064-91065436CD4C}"
+EndProject
+Global
```

Comment:
Solution file is missing reference to the `editorconfig` for the IDE to pick it up.
https://github.com/microsoft/kiota/blob/468a319ac5ba3ce2b7f09962e2402b1df276df07/http/dotnet/httpclient/Microsoft.Kiota.Http.HttpClientLibrary.sln#L8